tests/browser_test.py

```diff
@@ -1,168 +1,92 @@
-    )
-    assert account_creds_pass, "Account credentials password entry not found"
-    assert account_creds_pass.is_displayed()
-    account_creds_pass.send_keys(auth[1])
-
-    # sign in
-    sign_in_button: WebElement = selenium.find_element(By.CLASS_NAME, "btn-info")
-    assert sign_in_button, "Sign in button not found"
-    assert sign_in_button.text == "Sign in using Kinto Account Auth"
-    assert sign_in_button.is_displayed()
-    sign_in_button.click()
-
-    # determine if successfully logged in to admin home page
-    try:
-        server_info: WebElement = selenium.find_element(
-            By.XPATH,
-            "//div[@class='card-header' and contains(., 'Server information')]",
-        )
-        assert server_info, "Server information not found"
-        assert server_info.text == "Server information"
-        assert server_info.is_displayed()
-    except NoSuchElementException:
-        pytest.fail("Login was unsuccessful")
+    if not keep_existing:
+        setup_client.delete_records()
+
+
+def test_login_and_submit_review(
+    server, 
+    page, 
+    editor_auth, 
+    source_bucket, 
+    source_collection,
+    setup_auth,
+    skip_server_setup,
+    keep_existing
+):
+    # load login page
+    page.goto(f"{server}/admin/")
+    expect(page).to_have_title(re.compile("Remote Settings"))
+
+    # login
+    page.get_by_label("Kinto Account Auth").click()
+    txtUsername = page.get_by_label(re.compile("Username"))
+    txtPassword = page.get_by_label(re.compile("Password"))
+    txtUsername.fill(editor_auth[0])
+    txtPassword.fill(editor_auth[1])
+    page.get_by_text(re.compile("Sign in using Kinto Account Auth")).click()
+
+    # verify home page loaded
+    expect(page.get_by_text("project_name")).to_be_visible()
+    expect(page.get_by_text("project_version")).to_be_visible()
+    expect(page.get_by_text("http_api_version")).to_be_visible()
+    expect(page.get_by_text("project_docs")).to_be_visible()
+
+    # navigate to test collection
+    page.click('[href="#/buckets/main-workspace/collections/integration-tests/records"]')
+    expect(page.get_by_text("Records of main-workspace/integration-tests")).to_be_visible()
+    
+    # create a record
+    page.get_by_text("Create record").first.click()
+    page.get_by_label("JSON record").fill('{"prop": "val"}');
+    page.get_by_text("Create record").click()
+
+    # request a review
+    page.get_by_text("Request review...").first.click()
+    page.get_by_placeholder("Comment...").last.fill("Review comment")
+    page.get_by_text("Request review").last.click()
+
+    # verify that we are in-progress for review
+    expect(page.locator(".bs-wizard-step.complete").first).to_contain_text("Work in progress")
+    expect(page.locator(".bs-wizard-step.active").first).to_contain_text("Waiting review")
+    expect(page.locator(".bs-wizard-step.disabled").first).to_contain_text("Approved")
+
+# TODO: reviewer test scenario - login to approve pending request
```

Comment:
Is the plan to address this TODO in another PR? If so, let's turn it into an issue and remove this line